experimental/packages/otlp-transformer/README.md

```diff
@@ -19,43 +19,60 @@ To get started you will need to install a compatible OpenTelemetry API.
 npm install @opentelemetry/api
 ```
 
+### Experimental Features and Entrypoint
+
+This package is currently marked as experimental. New minor versions may include breaking changes, regardless of
+the entrypoint used. Once the package is marked as stable, only the `experimental/` entrypoint will break in
+minor versions of the package. If you use experimental features it is therefore recommended to pin the package or use
+to depend on a tilde-version.
+
 ### Serialize Traces/Metrics/Logs
 
 This module exports serializers to serialize traces, metrics and logs from the OpenTelemetry SDK into protocol buffers
```

Comment:
```suggestion
This package exports serializers to serialize traces, metrics and logs from the OpenTelemetry SDK into protocol buffers
```

Reply:
FYI in my initial attempt I broke up these sections a bit, if that's useful feel free to steal it from https://github.com/open-telemetry/opentelemetry-js/pull/5259/files#diff-427df6e9ac7ef1fa78e3c42356b23b30b56bfd11ca774f90c105a9d43cbd7ac8R38-R112

Reply:
Thanks for the suggestion, applied in https://github.com/open-telemetry/opentelemetry-js/pull/5263/commits/da71c637defeceead5192fd1944646e4187d7292 :)
I think we can leave one big entry for the examples for now. Since we dropped the `create*ExportRequest()` functions from the exports, there's actually nothing much to describe anymore.